### DISCUSSION TITLE
Undocumented Option indexAttribute

### DISCUSSION THREAD

**Discussion by Ben-Pusey-Bentley** [TanStack/virtual#740] [Q&A] [answered]
I've been implementing a dynamic virtualizer, and I wanted to use a different custom attribute for `data-index`. I found an option for `useVirtualizer` that does what I want, called [`indexAttribute`](https://github.com/TanStack/virtual/blob/3562e0640d0c7dbd688d2de96d438135b90a2e17/packages/virtual-core/src/index.ts#L317). The issue is that there is no documentation at all for this option, there are no examples that use it, and I can't find any discussion about this option at all. Is it safe to use this option?

**Comment by piecyk** [comment, marked as answer]
@Ben-Pusey-Bentley ooo for sure we need to documentation for it, but yes it safe to use.
